### PR TITLE
[Doc] Clarify the meaning of fee delta not being a fee rate in prioritisetransaction RPC

### PR DESCRIPTION
**Pull request by janbraiins**
Hi,

I have faced some confusion among our developers considering this being a fee rate. Would you consider including this tiny doc update?

Best regards,

Jan Capek

### CODE REVIEW

**Comment by MeshCollider** [comment]
Seems like a sensible addition, nice first contribution thanks :)

**Comment by MarcoFalke** [comment]
utACK c198dc00e1611b43d031920e7f9912fc25a359c3

File: src/rpc/mining.cpp [hunk 1]
```diff
@@ -240,6 +240,7 @@ UniValue prioritisetransaction(const JSONRPCRequest& request)
             "2. dummy          (numeric, optional) API-Compatibility for previous API. Must be zero or null.\n"
             "                  DEPRECATED. For forward compatibility use named arguments and omit this parameter.\n"
             "3. fee_delta      (numeric, required) The fee value (in satoshis) to add (or subtract, if negative).\n"
+            "                  Note, that this value is not a fee rate. It is a value to modify absolute fee of the TX.\n"
```
Comment (promag) [inline src/rpc/mining.cpp]: Instead of adding a new line, how about improving the description above (which IMO would be enough)?

**Comment by jnewbery** [comment]
utACK c198dc00e1611b43d031920e7f9912fc25a359c3

**Pull request merged**